lib/rules.json

```diff
@@ -1787,6 +1903,7 @@
                     "divClassHead": "The front matter <span class=\"rfc2119\">must</span> appear at the beginning of the body of the document, within &lt;div class=\"head\"&gt;. There is one exception to that requirement: the <code>hr</code> element after the copyright <span class=\"rfc2119\">may</span> appear inside or after the <code>div</code> element. Editors <span class=\"rfc2119\">should not</span> include other information in this section.",
                     "logo": "The document <span class=\"rfc2119\">must</span> include a link to the W3C logo identified below. The URI used to identify the logo <span class=\"rfc2119\">must</span> be absolute. <div class=\"boilerplate\"> <a href=\"https://www.w3.org/\"> <img alt=\"W3C\" src=\"https://www.w3.org/StyleSheets/TR/2016/logos/W3C\" height=\"48\" width=\"72\"> </a> </div> <div class=\"source\"> <span style=\"font-style: italic\">Include this source code</span>:<br><code>&lt;a href=\"https://www.w3.org/\"&gt;&lt;img height=\"48\" width=\"72\" alt=\"W3C\" src=\"https://www.w3.org/StyleSheets/TR/2016/logos/W3C\"/&gt;&lt;/a&gt;</code></div>",
                     "title": "The document's title <span class=\"rfc2119\">must</span> be in the <code>title</code> element and in an <code>h1</code> element.",
+                    "h3CrType": "The CR document's head <span class=\"rfc2119\">must</span> have a <code>h3</code> element. For Candidate Recommendation Draft, it's <pre xml:space=\"preserve\">&lt;h3&gt;Draft&lt;/h3&gt;</pre>For Candidate Recommendation Snapshot, it's <pre xml:space=\"preserve\">&lt;h3&gt;Snapshot&lt;/h3&gt;</pre>",
```

Comment:
I suggest you have the instruction for CR snapshot here but overwrite it for the profile CRD to be more specific on what is expected for both profiles.